Assets/MRTK/SDK/Features/UX/Scripts/Gaze.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Commenting on this for ToggleGazeSource.cs: I think it should be removed from the prefab instead of moved to the SDK. It's a pretty specific example script at the moment, and also isn't in use after https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7902

Reply:
done